constants.xml

```diff
@@ -48,9 +48,11 @@
         <FULL_DATASET_MINE>false</FULL_DATASET_MINE>
     </options>
     <smart_contract>
-        <SCILLA_PATH></SCILLA_PATH>
+        <SCILLA_ROOT></SCILLA_ROOT>
+        <SCILLA_BINARY>bin/scilla-runner</SCILLA_BINARY>>
```

Comment:
@kaikawaliu  an error with the closing tag `</SCILLA_BINARY>>`, should be `</SCILLA_BINARY>`